swarm/storage/mru/request.go

```diff
@@ -50,39 +42,14 @@ type updateRequestJSON struct {
 	Signature string `json:"signature,omitempty"`
 }
 
-// SignedResourceUpdate contains signature information about a resource update
-type SignedResourceUpdate struct {
```

Comment:
`SignedResourceUpdate` is moved to `signedupdate.go`, its own file. No new code.